pkgs/top-level/release-alternatives.nix

```diff
@@ -0,0 +1,91 @@
+{ pkgsFun ? import ../..
+, lib ? import ../../lib
+, supportedSystems ? ["x86_64-linux"]
+, allowUnfree ? false }:
+
+let
+
+  # called BLAS here, but also tests LAPACK
+  blasUsers = [
+    # "julia_07" "julia_10" "julia_11" "julia_13" "octave" "octaveFull"
+    "fflas-ffpack" "linbox" "R" "ipopt" "hpl" "rspamd" "octopus"
+    "sundials" "sundials_2" "superlu" "suitesparse_5_3" "suitesparse_4_4"
+    "suitesparse_4_2" "scs" "scalapack" "petsc" "magma" "cholmod-extra"
+    "arpack" "qrupdate" "libcint" "iml" "globalarrays" "arrayfire" "armadillo"
+    "xfitter" "lammps" "plink-ng" "quantum-espresso" "siesta"
+    "siesta-mpi" "shogun" "calculix" "csdp" "getdp" "giac" "gmsh" "jags"
+    "lammps" "lammps-mpi"
+
+    # requires openblas
+    # "caffe" "mxnet" "flint" "sage" "sageWithDoc"
```

Comment:
I don't think `flint` and `sage` necessarily require openblas. But since you added the comment I'm assuming you tested them. Maybe I'll have time to look into that once this is merged.